diff --git a/tests/check/Makefile.am b/tests/check/Makefile.am
index a963967..d786281 100644
--- a/tests/check/Makefile.am
+++ b/tests/check/Makefile.am
@@ -165,3 +165,13 @@ utils_binadded_SOURCES = \
 	testutils.c \
 	testutils.h \
 	utils/binadded.c
+
+nemotestsbindir = /opt/tests/farstream/bin
+nobase_nemotestsbin_PROGRAMS = $(check_PROGRAMS)
+
+farstream-tests.list:
+	$(AM_V_GEN)echo $(check_PROGRAMS) > $@
+
+nemotestsdatadir = /opt/tests/farstream/data
+nobase_nemotestsdata_DATA = $(EXTRA_DIST) farstream-tests.list
+
